projects/storefrontapp-e2e-cypress/cypress/integration/regression/checkout-as-persistent-user.e2e-spec.ts

```diff
@@ -1,7 +1,53 @@
 import * as checkoutAsPersistentUser from '../../helpers/checkout-as-persistent-user';
 import { formats } from '../../sample-data/viewports';
+import { config, login, setSessionData } from '../../support/utils/login';
 
 function checkoutAsPersistentUserTest() {
+  const username = 'test-user-cypress@ydev.hybris.com';
+  const password = 'Password123.';
+  const firstName = 'Test';
+  const lastName = 'User';
+  const titleCode = 'mr';
+
+  it('should register user if not exist', () => {
```

Comment:
We seem to have the same code elsewhere in the e2e utils ( register user if does not exist ).
Would there be a way to call that?

Reply:
We can improve this in a future iteration.